### PR TITLE
MPDX-7072- Insert data into ExpectedMonthlyTotal

### PR DESCRIPTION
**Pull request by rathclayton1**

### CODE REVIEW

File: src/components/Reports/ExpectedMonthlyTotalReport/Table/ExpectedMonthlyTotalReportTable.tsx [hunk 1]
```diff
@@ -17,70 +17,61 @@ import {
...
-  const total = () => {
-    return 3920;
-  };
-
   const showTotalPartners = () => {
     if (visible) {
-      return t(' Show ' + totalPartners() + ' Partners');
+      return t(
+        ' Show ' + data.length + ` Partner${data.length > 1 ? 's' : ''}`,
+      );
```
Comment (reldredge71) [inline src/components/Reports/ExpectedMonthlyTotalReport/Table/ExpectedMonthlyTotalReportTable.tsx:47]:
```suggestion
      return  data.length > 1 ? t('Show {{partnerCount}} Partners', { partnerCount: data.length }) : t('Show 1 Partner')
```
@OzzieOrca How does this look to you?

File: src/components/Reports/ExpectedMonthlyTotalReport/Table/ExpectedMonthlyTotalReportTable.tsx [hunk 2]
```diff
@@ -17,70 +17,61 @@ import {
...
     }
   };
-  return (
+
+  const isEmpty = data.length === 0;
```
Comment (reldredge71) [inline src/components/Reports/ExpectedMonthlyTotalReport/Table/ExpectedMonthlyTotalReportTable.tsx:51]: Actually you can use this variable to help with the logic for `showTotalPartners`.

File: src/components/Reports/ExpectedMonthlyTotalReport/Table/ExpectedMonthlyTotalReportTable.test.tsx [hunk 3]
```diff
@@ -2,18 +2,53 @@ import React from 'react';
...
+      <ExpectedMonthlyTotalReportTable
+        accountListId={'abc'}
+        title={'Donations So Far This Month'}
+        data={data.expectedMonthlyTotalReport.received.donations}
+        donations={true}
+        total={0}
+        currency={'USD'}
+      />
+    </ThemeProvider>,
+  );
```
Comment (reldredge71) [inline src/components/Reports/ExpectedMonthlyTotalReport/Table/ExpectedMonthlyTotalReportTable.test.tsx:49]: Could you add a test for `donations={false}`?

**Review by reldredge71 — CHANGES_REQUESTED** [review]

**Review by reldredge71 — APPROVED** [review]
looks good!

File: src/components/Reports/ExpectedMonthlyTotalReport/Table/ExpectedMonthlyTotalReportTable.tsx (hunk 1 above)
Comment (OzzieOrca) [reply src/components/Reports/ExpectedMonthlyTotalReport/Table/ExpectedMonthlyTotalReportTable.tsx]: I was going to suggest i18next's plural feature but if we're extracting text directly from the source code, I'm not sure how you'd specify other keys... This is fine.

**Pull request merged**